src/test/java/LoginTests.java

```diff
@@ -15,7 +15,7 @@ public static void LoginEmptyEmailPasswordTest () throws InterruptedException {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226166984-72fd2bb9-0d68-46a4-b0c4-f7d293c480f1.png)

Hint

 After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

 ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);


Below are the list of available and most commonly used arguments for ChromeOptions class

**start-maximized:** Opens Chrome in maximize mode
**incognito:** Opens Chrome in incognito mode
**headless:** Opens Chrome in headless mode
**disable-extensions:** Disables existing extensions on Chrome browser
**disable-popup-blocking:** Disables pop-ups displayed on Chrome browser
**make-default-browser:** Makes Chrome default browser
**version:** Prints chrome browser version
**disable-infobars:** Prevents Chrome from displaying the notification ‘Chrome is being controlled by automated software
![image](https://user-images.githubusercontent.com/104761482/226167059-7fe03827-bf06-45da-9a67-053cb55353a3.png)

![image](https://user-images.githubusercontent.com/104761482/226167109-48e7065e-7129-4eae-bc9f-2656d57bc3d9.png)

![image](https://user-images.githubusercontent.com/104761482/226167121-72b0fa4e-971f-44a2-98e8-3f041fe49860.png)

![image](https://user-images.githubusercontent.com/104761482/226167131-43beee9b-1d51-4978-a7a9-cd3ca00c93e1.png)

Reply:
I copied everything you added but I'm getting a "cannot find symbol class ChromeOptions" error message.

![LoginTests](https://user-images.githubusercontent.com/126837982/226199613-86483fa0-110f-4708-a895-51fa2bab71a4.jpg)

Reply:
Hello

Please, re-submit homework 15 for review